analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -337,6 +337,18 @@ End Sub
 Private Sub WithFunc(Arg As Func(Of Integer))
 End Sub");
 
+    [TestMethod]
+    public void Analyze_TopLevelStatements() =>
+        VerifyTopLevelStatements(@"
+string s = null; // Noncompliant {{Message for SMain}}
+LocalMethod(s);
+
+void LocalMethod(string s)
+{
+    s = null; // Noncompliant {{Message for SMain}}
+}
+");
```

Comment:
This shouldn't be here. That's a test for local methods.
```suggestion
```

Reply:
Then it needs a different name and test case near the other one related to local functions